### PR TITLE
Add halow ap config

### PR DESCRIPTION
**Pull request by jr1221**
This PR adds the configuration code for AP mode using the ALFA Hat.  Most of the changes were splitting STA and AP files such that one could be installed without the other, but they can coexist as long as they arent run at the same time.

### CODE REVIEW

File: odysseus/halow_setup/README.md [hunk 1]
```diff
@@ -56,38 +57,181 @@ The following command line items are run inside the pi terminal.  If you reboot
...
+   ssid="NER_Halow"
+   scan_ssid=1
+   key_mgmt=NONE
+   scan_freq= 2412
+   freq_list= 2412
+}
+p2p_disabled=1
+ignore_old_scan_res=1
+```
+Now set a static IP to ensure stability and easy connection to AP-side programs (ex. Argos).
```
Comment (nwdepatie) [inline odysseus/halow_setup/README.md:84]: This all feels like A LOT for a readme, this stuff should probably go in our confluence, especially since it is private and you can format everything easier. Confluence uses markdown iirc so you can just copy paste a lot of this stuff over

File: odysseus/halow_setup/install/ap/nrc-autostart-ap.service [hunk 2]
```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Control nrc using the pyton scripts provided (AP mode).
```
Comment (nwdepatie) [inline odysseus/halow_setup/install/ap/nrc-autostart-ap.service:2]: `pyton` lol

File: odysseus/halow_setup/setup_system.sh [hunk 3]
```diff
@@ -50,8 +50,8 @@ mv /etc/wpa_supplicant/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant.co
 
 
 echo "7. Enabling i2c, mac80211"
-output_i2c="$(cat "/etc/modules" | grep -e "i2c-dev")"
-output_mac="$(cat "/etc/modules" | grep -e "mac80211")"
+output_i2c="$(grep -e "i2c-dev" < "/etc/modules")"
```
Comment (nwdepatie) [inline odysseus/halow_setup/setup_system.sh:53]: uh wait why do we need to find an I2C device driver? Not bad just curious

**Review by nwdepatie — CHANGES_REQUESTED** [review]
TLDR you should def move some of the documentation to confluence, dm me for write credentials

File: odysseus/halow_setup/setup_system.sh (hunk 3 above)
Comment (jr1221) [reply odysseus/halow_setup/setup_system.sh:53]: I was wondering the same thing I put this in the README: `TODO: Does nrc actually use i2c?`
I tried Halow without it and it worked fine, I probably originally was basing my decision on this: https://docs.alfa.com.tw/Product/AHPI7292S/30_Technical_Details/
The i2c interface does relay some data to the pi, but that data is uneeded and it not userspace, so defintely not needed.

I removed that code.

**Review by nwdepatie — CHANGES_REQUESTED** [review]
big PR lol

**Review by nwdepatie — APPROVED** [review]
incorrectly requested changes my bad
